src/settings.cpp

```diff
@@ -829,9 +829,9 @@ directoryAddress = GetIniSetting ( IniXMLDocument, "server", "centralservaddr",
 
     // directory type
     // CServerListManager defaults to AT_NONE
-    // Server GUI defaults to AT_DEFAULT
+    // Server GUI also defaults to AT_NONE (previously AT_DEFAULT, when registration was controlled by a checkbox)
```

Comment:
I'd delete the comment entirely now - I think it's probably more confusing (trying to decypher past history).  It also makes the comment on the next line easier to follow.  (With less distraction.)

Reply:
OK, will do.